test/test_consistency.jl

```diff
@@ -0,0 +1,44 @@
+function check_quadratic_model(model, quadraticmodel)
+    rtol  = 1e-8
+    # @assert typeof(quadraticmodel) <: NLPModels.QuadraticModel
+    @assert quadraticmodel.meta.nvar == model.meta.nvar
+    @assert quadraticmodel.meta.ncon == model.meta.ncon
+  
+    x = [-(-1.0)^i for i = 1:quadraticmodel.meta.nvar]
+  
+    @assert isapprox(obj(model, x), obj(quadraticmodel, x), rtol=rtol)
+
+    f, g = grad(model, x)
```

Comment:
Does `grad()` return two things here? Did you mean `objgrad()`?

Reply:
Well noticed. I changed it.